modules/enet/networked_multiplayer_enet.cpp

```diff
@@ -659,6 +682,7 @@ void NetworkedMultiplayerENet::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("create_server", "port", "max_clients", "in_bandwidth", "out_bandwidth"), &NetworkedMultiplayerENet::create_server, DEFVAL(32), DEFVAL(0), DEFVAL(0));
 	ClassDB::bind_method(D_METHOD("create_client", "ip", "port", "in_bandwidth", "out_bandwidth"), &NetworkedMultiplayerENet::create_client, DEFVAL(0), DEFVAL(0));
 	ClassDB::bind_method(D_METHOD("close_connection"), &NetworkedMultiplayerENet::close_connection);
+	ClassDB::bind_method(D_METHOD("disconnect_peer"), &NetworkedMultiplayerENet::disconnect_peer, DEFVAL(false));
```

Comment:
I know, I'm an awful reviewer, this should be: `ClassDB::bind_method(D_METHOD("disconnect_peer", "id", "now"), ...` (notice the parameters string to get better names in the editor help, instead of `arg0` and `arg1`)